src/components/blocks/projects-two-columns.svelte

```diff
@@ -26,13 +26,20 @@
         <div class="flex first:pr-8 first:border-r last:pl-8 py-6">
           {#each measurement.measurements || [] as measurements}
             <div class="flex flex-col mr-3 w-24">
-              <p class="text-xs text-foreground-secondary font-medium uppercase">
-                {measurements.title}
-              </p>
-              <div class="flex">
-                <p class="md:text-lg text-3xl font-semibold">{measurements.value}%</p>
-                <img class="max-h-2.5 mt-2 ml-1.5" src={triangle} alt={measurements.title} />
-              </div>
+              <Popover variant={'fit-content'}>
+                <div slot="target">
+                  <p class="text-xs text-foreground-secondary font-medium uppercase">
+                    {measurements.title}
```

Comment:
this can be blank or undefined careful
https://drops.significa.co/Screen-Recording-2024-03-28-16-05-39-1711641939